TileStache/Goodies/VecTiles/transform.py

```diff
@@ -3185,3 +3185,35 @@ def elevation_to_meters(shape, props, fid, zoom):
 
     props['elevation'] = _to_float_meters(elevation)
     return shape, props, fid
+
+
+def add_is_bicycle_route(shape, props, fid, zoom):
+    """
+    If the props contain a bicycle_network tag or cycleway, it should
+    have an is_bicycle_route boolean.
+    """
+    props.pop('is_bicycle_route', None)
+    if ('bicycle_network' in props or
+            'cycleway' in props or
+            'cycleway_left' in props or
+            'cycleway_right' in props or
+            'cycleway_both' in props):
+        props['is_bicycle_route'] = 'yes'
+    return shape, props, fid
+
+
+def normalize_cycle_left_right(shape, props, fid, zoom):
```

Comment:
If `cycleway_both` is present but `cycleway` is not present, then rename `cycleway_both` to `cycleway`. If both are present, drop `cycleway_both`.